### PR TITLE
fixing dotenv behavior

### PR DESCRIPTION
**Pull request by wjcunningham7**
- [ ] I have added the tests to cover my changes.
- [ ] I have updated the documentation and CHANGELOG accordingly.
- [ ] I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by AlejandroEsquivel** [comment]
taking a look now

**Comment by AlejandroEsquivel** [comment]
taking a look now

File: covalent/_shared_files/config.py [hunk 1]
```diff
@@ -209,7 +217,7 @@ def set(self, key: str, value: Any) -> None:
         Returns:
             None
         """
-        set_key(find_dotenv(CONFIG_FILE_NAME), key, str(value))
+        set_key(self.config_file, key, str(value))
```
Comment (wjcunningham7) [inline covalent/_shared_files/config.py:220]: @AlejandroEsquivel I think this was the main offending line

Comment (AlejandroEsquivel) [reply covalent/_shared_files/config.py:220]: yep it was, it was a bit obscure what find_dotenv was doing under the hood

**Review by AlejandroEsquivel — APPROVED** [review]
Looks good to me, I think there can also be a change to the `.env.example` file to remove the legacy conf values however I think that can be done in a subsequent PR once we verify everything is working & we remove the commented out code as well.

**Pull request merged**